src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int MAX_RANDOM = 100;
+    private static final ColorSupplier COLOR_SUPPLIER = new ColorSupplier();
```

Comment:
```suggestion
    private static final ColorSupplier colorSupplier = new ColorSupplier();
```

because this is object